crates/bevy_window/src/system_cursor.rs

```diff
@@ -89,7 +89,7 @@ use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
     reflect(Serialize, Deserialize)
 )]
 #[reflect(Debug, PartialEq, Default)]
-pub enum CursorIcon {
+pub enum SystemCursorIcon {
```

Comment:
Not relevant to this PR, but I've had annoyances with how many of these are unavailable on macOS. It might be worth noting their limitation in the docs.

Reply:
Yeah that is probably a different issue.